DigitalLearningSolutions.Web/Views/Support/RequestSupportTicket/SupportTicketSummaryPage.cshtml

```diff
@@ -0,0 +1,88 @@
+﻿@using DigitalLearningSolutions.Data.Models.Support
+@using DigitalLearningSolutions.Web.Models.Enums
+@using DigitalLearningSolutions.Web.ViewModels.Support.RequestSupportTicket;
+@model SupportSummaryViewModel
+@{
+    var errorHasOccurred = !ViewData.ModelState.IsValid;
+}
+<a class="nhsuk-button nhsuk-button--secondary" role="button" asp-controller="RequestSupportTicket" asp-action="RequestAttachment">
```

Comment:
These Back links should use the [NHS Design System back-link component](https://service-manual.nhs.uk/design-system/components/back-link), as per wireframes.

Reply:
This has been implemented and changes are pushed. Please verify.